app.js

```diff
@@ -0,0 +1,103 @@
+import products from './Data/data.js';
+
+const productsData = products.slice();
+
+export function findById(items,id) {
+    for (let i = 0; i <items.lenghth; i++) {
+        const items = items [i];
+        if (items.id === id) {
+            return items;
+        }
+    }   
+}
+
+let totalVotes = 0;
+let productVotedResults = 0;
+
+const initializeState = () => {
+    totalVotes = 0
+    productVotesResults = [];
+};
+
+initializeState();
+
+const showThreeOptions = () => {
+    const option1 = randomProduct(productsData);
+    let option2 = randomProduct(productsData);
+    let option3 = randomProduct(productsData);
+    while (option1.id === option2.id
+        || option2.id === option3.id
+        || option1.id === option3.id
+    ){
+        options2 = randomProduct(productsData)
+        option3 = randomProduct(productsData)
+    }
+
+    const radio1 = document.getElementById('option1')
+    const radio2 = document.getElementById('option2')
+    const radio3 = document.getElementById('option3')
+    const radio1Span = document.getElementById('option1span');
+    const radio2Span = document.getElementById('option2span');
+    const radio3Span = document.getElementById('option3span');
```

Comment:
your HTML has the id's of these the same as for radio1, radio2, and radio3 so these are coming back null